client/gutenberg/editor/controller.js

```diff
@@ -198,7 +198,13 @@ export const post = ( context, next ) => {
 	} );
 
 	if ( config.isEnabled( 'calypsoify/iframe' ) ) {
-		context.primary = <CalypsoifyIframe postId={ postId } postType={ postType } />;
+		context.primary = (
+			<CalypsoifyIframe
+				postId={ postId }
+				postType={ postType }
+				duplicatePostId={ duplicatePostId }
```

Comment:
I get this warning:

```
Warning: Failed prop type: Invalid prop `duplicatePostId` of type `string` supplied to `CalypsoifyIframe`, expected `number`.
```

`parseInt` if not null?

Reply:
Nice catch, added a validation check in fe5db34 so values like ?jetpack-copy or ?jetpack-copy=foo won't emit a js warning in console.